packages/react-server/core/ReactServerAgent.js

```diff
@@ -10,8 +11,14 @@ function makeRequest (method, url) {
 	return new Request(method, url, API.cache());
 }
 
+const BUNDLE_CACHE     = {};
```

Comment:
Very nitpicky thought: we currently use "bundle" to talk about the static JS bundles, so maybe its worth using a different term here, or a more specific one like `DATA_BUNDLE_CACHE`. 

As always, feel free to ignore.